python/cudf/cudf/tests/test_orc.py

```diff
@@ -686,12 +686,12 @@ def test_orc_bool_encode_fail():
     # Should throw instead of generating a file that is incompatible
     # with other readers (see issue #6763)
     with pytest.raises(RuntimeError):
-        fail_df.to_orc("should_throw.orc")
+        fail_df.to_orc(tmpdir.join("should_throw.orc"))
```

Comment:
@galipremsagar instead of a `tmpdir`, can we just write / read from a `BytesIO` object here?

Reply:
Yeah we can do this too.

```suggestion
        fail_df.to_orc(BytesIO())
```